doc/tutorials/application-integration.md

```diff
@@ -205,39 +200,39 @@ For more information about that, continue reading
 
 ## Specification ##
 
-Now, we have a fully working configuration system without any hard-coded
-information (such as configuration files). So we already gained something.
+Now, we have a fully working configuration system without any hardcoded
```

Comment:
All forms seem to be acceptable: https://en.wikipedia.org/wiki/Hard_coding

Why change it?

Does the tool allow automatic transformation so that we have it easier in future when we change it?